CODE_OF_CONDUCT.md

```diff
@@ -1,25 +1,22 @@
-# Autocrypt Code of Conduct
+# Autocrypt Community Code of Conduct
 
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in the Autocrypt
-community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
+contributors and maintainers strive to making participation in the Autocrypt
+community a pleasurable, harassment-free experience. 
 
 ## Our Standards
 
-Examples of behavior that contributes to creating a positive environment include:
+Examples of behavior that we think contributes to creating a positive environment include:
```

Comment:
"we believe"?

Reply:
just stroke the "we X", it's not really needed i guess.